src/Razor/src/Microsoft.VisualStudio.Editor.Razor/EditorSettingsManager.cs

```diff
@@ -30,42 +30,3 @@ public EditorSettingsChangedEventArgs(EditorSettings settings)
 
     public EditorSettings Settings { get; }
 }
-
-public sealed class EditorSettings : IEquatable<EditorSettings>
```

Comment:
Can you just move the rest of this file back too? `EditorSettingsManager` is used by Web Tools, and there doesn't seem to be much point in keeping it here and forcing them to update.

(also can you put them in their own files when you do? :P)